README.md

```diff
@@ -124,6 +121,9 @@ try {
 * [AWS SDK Bundle for Symfony][mod-symfony]
 * [Amazon SNS Message Validator for PHP][sns-validator] - SNS validator without requiring SDK
 * [Guzzle Version 6][guzzle-docs] – PHP HTTP client and framework
+* For **the previous** version 2 of the SDK:
```

Comment:
You're totally right, i don't know why i was using such poor _Engrish!_ 
550a325